lib/index.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as bodyParser from 'body-parser';
+import * as express from 'express';
+import * as _ from 'lodash';
+
+import {NetworkManager} from './nm';
+import wifiApi from './routes/wifi';
+
+interface Route {
+	method: string;
+	handler: (nm: NetworkManager) => (req: any, res: any) => void;
+}
+
+async function HttpServer(port: number = 1337) {
```

Comment:
Should this return the created `app`, so that the server can be shut down gracefully?

Reply:
Totally, I see this is related to the README.md example too